app/views/browse/deployment.html

```diff
@@ -1,141 +1,266 @@
+                            </div>
+                          </dd>
+                          <dt>Paused:</dt>
+                          <dd>{{!!deployment.spec.paused}}</dd>
+                        </dl>
+                      </div>
+                      <div class="col-lg-6">
+                        <h3>Configuration</h3>
+                        <dl class="dl-horizontal left">
+                          <dt>Selectors:</dt>
+                          <dd>
+                            <selector selector="deployment.spec.selector"></selector>
+                          </dd>
+                          <dt>Strategy:</dt>
+                          <dd>{{deployment.spec.strategy.type | sentenceCase}}</dd>
+                          <dt ng-if-start="deployment.spec.strategy.rollingUpdate">Max Unavailable:</dt>
+                          <dd>
+                            {{deployment.spec.strategy.rollingUpdate.maxUnavailable}}
+                            <span data-toggle="tooltip"
+                              title="The maximum number of pods that can be unavailable during the update process."
+                              class="pficon pficon-help text-muted small"></span>
+                          </dd>
+                          <dt>Max Surge:</dt>
+                          <dd ng-if-end>
+                            {{deployment.spec.strategy.rollingUpdate.maxSurge}}
+                            <span data-toggle="tooltip"
+                              title="The maximum number of pods that can be created above the desired number of pods."
+                              class="pficon pficon-help text-muted small"></span>
+                          </dd>
+                          <dt>Min Ready:</dt>
+                          <dd>
+                            {{deployment.spec.minReadySeconds || 0}} sec
+                            <span data-toggle="tooltip"
+                              title="The minimum number of seconds for which a newly created pod should be ready without any of its containers crashing for it to be considered available."
+                              class="pficon pficon-help text-muted small"></span>
+                          </dd>
+                        </dl>
+                      </div>
+                    </div>
+                    <div class="row">
+                      <div class="col-lg-6">
+                        <h3>Template</h3>
+                        <pod-template
+                            pod-template="deployment.spec.template"
+                            images-by-docker-reference="imagesByDockerReference"
+                            builds="builds"
+                            detailed="true"
+                            add-health-check-url="{{ ({ group: 'extensions', resource: 'deployments' } | canI : 'update') ? healthCheckURL : '' }}">
```

Comment:
That's what this is doing. The pod template directive doesn't know what resource to check, so it has to be outside.